src/main/Entities/Participant.java

```diff
@@ -0,0 +1,32 @@
+package Entities;

```

Comment:
The package name should be in all lowercase: change to "entities" instead. I think this is how the Gradle project is setup.